e2e-tests/demand-backup-cloud/run

```diff
@@ -68,6 +68,71 @@ run_recovery_from_source() {
 	compare_mysql_cmd "select-1" "SELECT * from myApp.myApp;" "-h $cluster-pxc-2.$cluster-pxc -uroot -proot_password"
 }
 
+# If backup upload was started and failed for some reaason the cloud storage should be cleaned up during second try
+delete_backup_pod() {
+	local backup_name=$1
+	local max_retry=120
+
+	echo "Waiting for ${backup_pod} to become Running"
+	set -o xtrace
+	# retry=0
+	sleep 1
+	kubectl wait --for=jsonpath='{.status.phase}'=Running pod --selector=percona.com/backup-job-name=xb-${backup_name} --timeout=120s
+	# until kubectl get pods --selector=percona.com/backup-job-name=xb-${backup_name} -o jsonpath='{.items[].status.phase}' 2>/dev/null | grep -q -i 'Running'; do
+	# 	sleep 1
+	# 	echo -n .
+	# 	let retry+=1
+
+	# 	if [[ $retry -ge $max_retry ]]; then
+	# 		echo max retry count $retry reached. something went wrong with operator or kubernetes cluster
+	# 		exit 1
+	# 	fi
+	# done
```

Comment:
looks like `kubectl wait` does the job, please remove commented code

Reply:
Yes, this was first variant before coming up with `kubectl wait` solution.